app/validators/api/v2025_0_rc/hesa_trainee_detail_attributes/rules/funding_method.rb

```diff
@@ -63,6 +63,16 @@ def no_funding_method?
           def funding_type
             @funding_type ||= FUNDING_TYPES[funding_method]
           end
+
+          def academic_cycle
+            @academic_cycle ||= start_date.present? ? AcademicCycle.for_date(start_date) : AcademicCycle.for_date(Time.zone.now + ::Trainees::SetAcademicCycles::DEFAULT_CYCLE_OFFSET)
+          end
+
+          def start_date
+            start_date = trainee_attributes.trainee_start_date || trainee_attributes.itt_start_date
+            start_date = Date.parse(start_date) if start_date.present? && start_date.is_a?(String)
```

Comment:
Should we check that the date can be parsed similar to [valid_date_string?](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/models/api/v0_1/trainee_attributes.rb#L312)?